### ISSUE TITLE
file find errors after running benchmark

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#48]
```
What steps will reproduce the problem?
1. starting benchmark with its GUI button

What is the expected output? What do you see instead?
I dont get the results after the benchmark, only errors.

What version of the product are you using? On what operating system?
1.0.1, Windows XP SP3

Please provide any additional information below.

namebench.exe.log:
Traceback (most recent call last):
  File "libnamebench\tk.pyc", line 87, in run
  File "libnamebench\base_ui.pyc", line 92, in RunBenchmark
  File "libnamebench\base_ui.pyc", line 111, in DisplayHtmlReport
  File "libnamebench\better_webbrowser.pyc", line 77, in open
  File "webbrowser.pyc", line 61, in open
  File "libnamebench\better_webbrowser.pyc", line 58, in open
  File "libnamebench\better_webbrowser.pyc", line 33, in create_win32_http_cmd
WindowsError: [Error 2] The system cannot find the file specified
```

Original issue reported on code.google.com by `norb...@gmail.com` on 8 Dec 2009 at 7:05
- Merged into: #43

**Comment by GoogleCodeExporter** [comment]
```
Try namebench 1.0.2 - it should resolve these issues.

```

Original comment by `tstromb...@google.com` on 8 Dec 2009 at 7:32
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
Not sure what issue 43 has to do with this, as Firefox was clearly my default 
browser
according to system settings.

Problem seems to have been fixed in 1.0.2
```

Original comment by `mikewill...@gmail.com` on 8 Dec 2009 at 8:58
